### ISSUE TITLE
How to use v-dragPreview ? 

### ISSUE THREAD

**Issue by yandeqiang** [jenshaase/vue-react-dnd#3]
Hi: 
  Thanks you for this lib at first.

  I have two questions:
  * How to use v-dragPreview ? 
  * How to get enough doc?

thanks.

**Comment by jenshaase** [comment]
Hi,

> How to use v-dragPreview ?

Have a look a the CustomDragLayer example. This uses v-dragPreview: https://github.com/jenshaase/vue-react-dnd/blob/master/docs/examples/DragAround/CustomDragLayer/DraggableBox.vue

> How to get enough doc?

Sorry, can you rephrase the question. I don't know what you mean?

Best regards,
Jens

**Comment by yandeqiang** [comment]
Thank you for your reply.

Here is my code. 

 I want to display text by default, display image when dragging.

```vue

<template>
  <div>
    <div class="drag-element" v-dragSource v-bind:class="{dragging: isDragging}">
      <text>name</text>
      <img v-if="isDragging" class="drag-element-preview" v-dragPreview :src="element.dragPreview" alt="">
    </div>
  </div>
</template>

```

**Comment by jenshaase** [comment]
I haven't tried it myself but I think you should not nest the preview element inside the drag element.

Try this:
```html
<template>
  <div>
    <div class="drag-element" v-dragSource v-bind:class="{dragging: isDragging}">
      <text>name</text>
    </div>
    <img class="drag-element-preview" v-dragPreview :src="element.dragPreview" alt="">
  </div>
</template>
```
